app/domain/authentication/resource_restrictions/extract_resource_restrictions.rb

```diff
@@ -7,7 +7,7 @@ module ResourceRestrictions
       dependencies: {
         resource_restrictions_class: Authentication::ResourceRestrictions::ResourceRestrictions,
         get_restriction_from_annotation: Authentication::ResourceRestrictions::GetRestrictionFromAnnotation.new,
-        fetch_resource_annotations: Authentication::ResourceRestrictions::FetchResourceAnnotations.new,
+        fetch_resource_annotations_instance: Authentication::ResourceRestrictions::FetchResourceAnnotations.new,
```

Comment:
we don't need the `_instance` suffix. we don't do it anywhere else

Reply:
Becaouse name fetch_resource_annotations is caught

Reply:
by whom? can you please point me to it?